fhttp/http.go

```diff
@@ -1163,6 +1163,16 @@ func init() {
 	rand.Read(payload) // nolint:gas
 }
 
+// ChangeMaxPayloadSize is used to change max payload size and fill it with pseudorandom content
+func ChangeMaxPayloadSize(newMaxPayloadSizeKb int) {
```

Comment:
change init() to call this as well or ideally to do it only if not already done by flags

also wtb test